src/BenchmarkDotNet/Portability/RuntimeInformation.cs

```diff
@@ -155,6 +156,22 @@ internal static CpuInfo GetCpuInfo()
             return null;
         }
 
+        public static bool IsVistaAndAbove() => Version.Parse(RuntimeEnvironment.OperatingSystemVersion).Major >= 6;
```

Comment:
BenchmarkDotNet v0.11.0+ requires .NET Framework 4.6+. Windows XP doesn't support .NET Framework 4.6+ and .NET Core. So, we can assume that the Windows version is always vista or above.

Reply:
Removed the code